README.md

```diff
@@ -33,17 +38,24 @@ To initialize the client, you need a server address and a configuration key (and
 You must keep a reference to the created API client instance until the SDK UI terminates.
 
 ```swift
+import UIKit
 import NinchatSDKSwift
 import NinchatLowLevelClient
 
-self.ninchatSession = NINChatSession(configKey: configKey, queueID: queueID, environments: environments)
-self.ninchatSession.delegate = self
+var ninchatSession: NINChatSessionSwift!
+ninchatSession = NINChatSessionSwift(configKey: configKey, queueID: queueID, environments: environments)
+ninchatSession.siteSecret = ""
```

Comment:
Site secret is an internal thing - no external app developer ever has a site secret. (External SDK developers have them, though.) Let's not mention it anywhere, it will only confuse users.